data.json

```diff
@@ -1,120 +1,80 @@
 [
 	{
 		"id": "55b03479-78d9-4384-8509-9457e49a087a",
-
 		"businessRecipient": "Stable",
 		"from": "First Republic Bank",
 		"imageUrl": "https://s3-us-west-2.amazonaws.com/usestable.com-assets/mail/FRB1.png",
 		"individualRecipient": "Sarah Ahmad",
-		"timestamp": 1596585600000
+		"timestamp": 1696585600000
```

Comment:
nit: want to switch these to createdAt to be more in line with our data model today?